.cirrus.yml

```diff
@@ -1074,6 +1075,26 @@ win_installer_task:
     main_script: '.\contrib\cirrus\win-installer-main.ps1'
 
 
+windows_smoke_test_task:
+    name: "Windows Smoke Test"
+    alias: windows_smoke_test
+    # Don't run for multiarch/container image cirrus-cron job.
+    only_if: $CIRRUS_CRON != 'multiarch'
+    depends_on:
+      - "Windows Cross"
```

Comment:
Request: could you please change this to `- alt_build`? `cirrus-task-map` (the script used for generating CI flow diagrams) cannot grok "Windows Cross", and it doesn't look like that's even possible to fix.